test/UserTest.php

```diff
@@ -10,7 +10,13 @@ public function testSuccessCreateUserCardHtml1()
     {
         $user = new User(1,'fakeUsername', 'fakeHashedPass', 'fake bio', 'fakeAvatar.jpeg');
         $result = $user->createUserCardHtml();
-        $expected = '<div class="userCard"><img class="avatarImg" src="fakeAvatar.jpeg" alt="Profile Picture"><p class="cardUsernameText">fakeUsername</p></div>';
-        $this->assertEquals($expected, $result);
+        $this->assertEquals('<a class="profileLink" href="./userProfile.php?username=fakeUsername"><div class="userCard"><img class="avatarImg" src="./assets/imgs/avatars/fakeAvatar.jpeg" alt="Profile Picture"><p class="cardUsernameText">fakeUsername</p></div></a>', $result);
```

Comment:
not sure why this was changed from before but it is better for the expected output to be put into a variable so that this line is more readable like this:
```php
$this->assertEquals($expected, $result);
```

Reply:
We probably messed up a merge, have changed these as requested, god.